src/components/Text.re

```diff
@@ -8,6 +8,47 @@ external make:
     ~accessible: bool=?,
     ~accessibilityHint: string=?,
     ~accessibilityLabel: string=?,
+    ~accessibilityRole: [@bs.string] [
+                          | // below are additional roles added for react-native-web https://github.com/necolas/react-native-web/blob/36dacb2052efdab2a28655773dc76934157d9134/packages/react-native-web/src/modules/AccessibilityUtil/propsToAccessibilityComponent.js#L13-L23
```

Comment:
You can see that the compiler is moving the comment up there (which sucks but that's a fact...)
What about instead keeping everything Alpha ordered with just `// web` after web only values?

Reply:
Or a general comment for the accessibility roles enumerating all the variants that apply to react-native-web. This is probably also better for tools that extract documentation from the source.

Reply:
@MoOx @jfrolich 
Thanks for the suggestion! :)
I've listed the strings in alphabetical order, and added comment about react-native-web specific roles.